src/seedu/addressbook/data/person/Person.java

```diff
@@ -83,5 +83,18 @@ public int hashCode() {
     public String toString() {
         return getAsTextShowAll();
     }
+    
+    /**
+     * Returns a concatenated version of the printable strings of each object. 
+     */
+    String getPrintableString(Printable... printables) {
+    	int i = 0;
+    	String concatenated = "";
+    	while (i < printables.length) {
+    		concatenated += printables[i];
```

Comment:
i think you need to call the individual `getPrintableString()` methods here, not just concatenate the objects.